sdk/python/kubeflow/training/api/training_client.py

```diff
@@ -133,6 +135,10 @@ def train(
             name: Name of the PyTorchJob.
             namespace: Namespace for the PyTorchJob. By default namespace is taken from
                 `TrainingClient` object.
+            labels: Labels for the Job. You can specify a dictionary as a mapping object
+                representing the labels.
+            annotations: Annotations for the Job. You can specify a dictionary as a
+                mapping object representing the annotations.
```

Comment:
Looks good to me, just a nit for consistency:
```suggestion
            labels: Labels for the PyTorchJob. You can specify a dictionary as a mapping object
                representing the labels.
            annotations: Annotations for the PyTorchJob. You can specify a dictionary as a
                mapping object representing the annotations.
```

Reply:
Actually, this API can be used for other types of job also (e.g. TFJob).

Reply:
Well, [every other arg in train()](https://github.com/kubeflow/trainer/blob/8488ab57fdd00a7f70fe04f2e63d7fc477afa762/sdk/python/kubeflow/training/api/training_client.py#L135-L146) refers to PyTorchJob, so I'd keep labels and annotation in line with that

Reply:
Oh, sorry this suggestion is for `train()` API.
Yeah, in V1 we use PyTorchJob with this API, so we should change it as @kramaranya suggested.

Reply:
@kramaranya @andreyvelich Thanks a lot for reviewing the PR, I have made the chnages accordingly.. Please check!

Reply:
@abhijeet-dhumal thank you! 
lgtm